docs/guides/run-jobs-session.mdx

```diff
@@ -72,19 +55,6 @@ with Session(backend=backend):
   estimator = Estimator()
   sampler = Sampler()
 ```
-  </TabItem>
-
-  <TabItem value="PrimV1" label="V1 primitives">
-``` python
-from qiskit_ibm_runtime import Session, Sampler, Estimator
-
-backend = service.least_busy(operational=True, simulator=False)
-with Session(backend=backend):
-  estimator = Estimator()
-  sampler = Sampler()
-```
-  </TabItem>
-</Tabs>
 
 When you start a session,  you must specify a QPU. This can be done by specifying its name or by passing a `backend` object.
```

Comment:
Specifying a QPU name is deprecated (the code snippet also doesn't do that). I thought this was fixed in the other PR 🤔

Reply:
I thought so too - I'll try merging main into this PR again